webrecorder/browsertrix
# Editable exclusion table cells

Enables users to edit exclusion table cells directly when creating and editing crawl templates. Resolves #376.

### Manual testing
1. Run `yarn start`
2. Create a new crawl template, interacting with "Exclusions" table in "Crawl Settings" section. Verify table cells are editable, new row can be added/removed, and errors show on submit attempt.
3. Save new crawl template. Verify exclusions are saved as expected on crawl detail view.
4. Click "Edit" in "Configuration" section. Update exclusions, verify exclusions are saved as expected.

### Screenshots
**New Crawl Template view:**
<img width="637" alt="Screen Shot 2022-11-21 at 2 54 59 PM" src="https://user-images.githubusercontent.com/4672952/203185183-1ea02ad1-6e4e-4a50-8bc2-1855cbffff51.png">

**Error states:**
<img width="629" alt="Screen Shot 2022-11-21 at 4 25 59 PM" src="https://user-images.githubusercontent.com/4672952/203185204-3134cc58-7829-4db2-bb45-a1a0475a25f8.png">
<img width="635" alt="Screen Shot 2022-11-21 at 4 26 05 PM" src="https://user-images.githubusercontent.com/4672952/203185210-9b03f29d-5325-4dec-9450-4495877837f4.png">
<img width="637" alt="Screen Shot 2022-11-21 at 4 26 15 PM" src="https://user-images.githubusercontent.com/4672952/203185212-3069ae6c-f6bd-4507-b1ef-d493087b3069.png">
